release/Dockerfile.in

```diff
@@ -1,16 +1,43 @@
 # Concatenated after ../Dockerfile to create the tgz
 FROM docker.io/fortio/fortio.build:v38 as stage
+ARG archs="amd64 arm64 ppc64le s390x"
+ENV archs=${archs}
+
 WORKDIR /stage
-COPY --from=release /usr/bin/fortio usr/bin/fortio
 #COPY --from=release /usr/share/fortio usr/share/fortio
 COPY docs/fortio.1 usr/share/man/man1/fortio.1
-RUN mkdir /tgz
-# Make sure the list here is both minimal and complete
-# we could take all of * but that adds system directories to the tar
-RUN tar cvf - usr/share/man/man1/fortio.1 usr/bin/fortio | gzip --best > /tgz/fortio-linux_x64-$(./usr/bin/fortio version -s).tgz
-COPY --from=build /go/src/fortio.org/fortio.exe usr/share/fortio/
-WORKDIR /stage/usr/share/fortio
-RUN zip -9 -r fortio_win_$(/stage/usr/bin/fortio version -s).zip fortio.exe && mv *.zip /tgz
+
+RUN mkdir -p /tgz usr/bin
+
+WORKDIR /go/src/fortio.org
+COPY . fortio
+# Check we still build with go 1.8 (and macos does not break)
+RUN make -C fortio official-build BUILD_DIR=/build OFFICIAL_BIN=../fortio_go_latest.mac GOOS=darwin GO_BIN=/usr/local/go/bin/go
+# Windows release, for now assumes running from fortio directory (static resources in .)
+RUN make -C fortio official-build BUILD_DIR=/build LIB_DIR=. OFFICIAL_BIN=/tmp/fortio.exe GOOS=windows
+# Linux per-architecture binaries building
+RUN sh -c \
+    'set -ex; for arch in ${archs}; do \
+       make -C fortio official-build-version BUILD_DIR=/build GOARCH=${arch} OFFICIAL_BIN=/tmp/fortio_${arch}; \
```

Comment:
Assuming it is at all possibel, we shouldn't have to rebuild here, the Dockerfile in parent dir should have built everything before the per arch from scratch gets created and we should be able to 
COPY --from=release /usr/bin/fortio usr/bin/fortio to get the binary from each of the multi arch?

Reply:
Hello, I apologize for the late reply. 

It cannot be done in a multiarch scenario this way.

When you build a multiarch image, you build N_ARCH images that will be packaged into a manifest list.

When building this way, copying a file from a previous stage means doing N_ARCH (copy) layers in N_ARCH separate "contexts/containers".

I know we are rebuilding two times the same binaries, but it is done:
1. to build a multiarch manifest list
2. Then, to (cross)-build 4 binaries in a single amd64 image and publish them in the release section of this repository

The alternative could be starting from the second point and building the multi-arch image later. I didn't do to not change more than what I already did till now, your previous workflow.

Reply:
Thanks, I need to read up a bit about multiarch

So it's not possible to address specifically or enumerate all the multi arch images built (to copy from)?

I appreciate greatly the contribution btw! it will be great to have, just trying to wrap my head around the effect on CI 

I probably need to take over your branch/fork to build the new build image and bump the version
( https://github.com/fortio/fortio/tree/master/release#how-to-change-the-build-image )

Reply:
we'll follow up; if at all possible in #550